admission_control/admission-control-service/src/upstream_proxy.rs

```diff
@@ -170,6 +176,13 @@ fn pick_peer(peer_info: &HashMap<PeerId, bool>) -> Option<PeerId> {
     None
 }
 
+fn count_active_peers(peer_info: &HashMap<PeerId, bool>) -> i64 {
```

Comment:
probably would be cleaner to return usize here and perform conversion to i64 on caller side